.github/release-drafter.yml

```diff
@@ -1,21 +0,0 @@
-name-template: Micronaut $NEXT_PATCH_VERSION
```

Comment:
why are you deleting this file in this PR? this is unrelated to Fixes https://github.com/micronaut-projects/micronaut-core/issues/7309 ?

Reply:
I restored it here: https://github.com/micronaut-projects/micronaut-core/pull/7308/commits/8a3964715a856258df475b85e3587670e3ee4b16